{
    "original_source": {
        "database(url)": "https://github.com/secureIT-project/CVEfixes",
        "software_system(CPE)": [
            "cpe:2.3:a:autojump_project:autojump:*:*:*:*:*:*:*:*",
            "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*",
            "cpe:2.3:o:debian:debian_linux:9.0:*:*:*:*:*:*:*",
            "cpe:2.3:o:debian:debian_linux:10.0:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/wting/autojump/commit/c763b2afadb188ab52849c21d43d2e8fe5b8800a",
        "version_end_excluding": {
            "autojump_project:autojump": "21.5.8",
            "debian:debian_linux": "unknown"
        },
        "path": {
            "file_path": "bin/autojump"
        }
    },
    "programming_language": "python",
    "licences": {
        "database": "MIT",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2013-2012"
        ],
        "CWE": [
            "CWE-269"
        ]
    }
}